src/Stoa.ts

```diff
@@ -774,21 +784,318 @@ class Stoa extends WebService
+                res.status(400).send(`Invalid value for parameter 'page': ${req.query.page.toString()}`);
+                return;
+            }
+            page = Number(req.query.page.toString());
+        }
+        else
+            page = 1;
+
+        this.ledger_storage.getBlockTransactions(field, value, limit, page)
+        .then((data: any) => {
+            if ((data === undefined)) {
+                res.status(500).send("Failed to data lookup");
+                return;
+              }
+            else if (data.tx.length === 0) {
+               return res.status(204).send(`The data does not exist. 'height': (${value})`);
+            }
+            else {
+                let tx: Array<IBlockTransactionElements> = [];
+                for (const row of data.tx) {
+                    tx.push(
     {
+                            height: JSBI.BigInt(row.block_height).toString(),
+                            tx_hash: new Hash(row.tx_hash, Endian.Little).toString(),
+                            amount: row.amount,
+                            type: row.type,
+                            fee: row.tx_fee,
+                            size: row.tx_size,
+                            time: row.time_stamp,
+                            sender_address: row.sender_address,
+                            receiver: JSON.parse(row.receiver),
+                        }
+                    );
+                }
+
+                let transactionList: IBlockTransactions = {
+                    tx,
+                    total_data: data.total_data
+                };
+               return res.status(200).send(JSON.stringify(transactionList));
+            }
+        })
+        .catch((err) => {
+            logger.error("Failed to data lookup to the DB: " + err);
+            res.status(500).send("Failed to data lookup");
+        });
+    }
+
+    /**
+     * GET /boa-stats
+     *
+     * Called when a request is received through the `/boa-stats` handler
+     *
+     * @returns Returns statistics of BOA coin.
+     */
+    private getBOAStats(req: express.Request, res: express.Response) {
+        
+        logger.http(`GET /boa-stats/`);
+
+        this.ledger_storage.getBOAStats()
+        .then((data: any) => {
+            if ((data === undefined)) {
+                res.status(500).send("Failed to data lookup");
+                return;
+              }
+           else if (data.length === 0) {
+               return res.status(204).send(`The data does not exist`);
+            }
+            else {
+                let boaStats: IBOAStats = {
+                    height: data[0].height,
+                    transactions: data[0].transactions,
+                    validators: data[0].validators,
+                    frozen_coin: 5283595, //static data because of unavailability of real data
+                    circulating_supply: 5283535,
+                    active_validators: 155055
```

Comment:
//FIX ME static data because of unavailability of real data

Reply:
updated